controllers/routes.js

```diff
@@ -64,6 +65,11 @@ const toplevel = {
             path: '/search',
             router: require('./search')
         },
+        {
+            path: '/user',
```

Comment:
These routes should be available in welsh so moving into routes rather than as a standalone import.